scripts/update_from_schema.js

```diff
@@ -0,0 +1,372 @@
+// When running this script add the path to the schema file to use as argv
```

Comment:
Need a couple of sentences about why is this script needed?  The `main` function shows it updates all the major pieces.  Why is it needed now?  When to use this script?